doc/sources/guide/packaging-osx.rst

```diff
@@ -1,82 +1,192 @@
+    > mkdir packaging
     > cd packaging
     packaging> git clone https://github.com/kivy/kivy-sdk-packager
     packaging> cd kivy-sdk-packager/osx
     osx> cp -a /Applications/Kivy.app ./Kivy.App
 
-This step above is important, you have to make sure to preserve the paths and permissions. A command like cp -rf will copy but make the app unusable and lead to error later on.
+  .. note::
+    - This step above is important, you have to make sure to preserve the paths and permissions.
+      A command like ``cp -rf`` will copy but make the app unusable and lead to error later on.
 
-Now all you need to do is to include your compiled app into the Kivy.app, simply run the following command::
+3. Now all you need to do is to include your compiled app into the Kivy.app. Simply run the following command::
 
-    osx> ./package-app.sh path/to/your/app
+    osx> ./package-app.sh /Applications/Kivy.app/Contents/Resources/kivy/examples/demo/touchtracer/
 
-This should copy Kivy.app to `yourapp.app` and include a compiled copy of your app into this package.
+  This copies Kivy.app to `touchtracer.app` and includes a compiled copy of the demo app into this package.
 
-That's it, your self contained package is ready to be deployed!
+4. That's it, your first self contained package is ready to be deployed!
+   You can now further customize your app as described bellow.
 
-when you double click this app you can see your app run.
 
+Shrink the app size
+^^^^^^^^^^^^^^^^^^^
 This is a pretty big sized app right now however you can simply remove the unneeded parts from this package.
 
-For example if you don't use Gstreamer, simply remove it from YourApp.app/Contents/Frameworks.
+For example if you don't use GStreamer, simply remove it from YourApp.app/Contents/Frameworks.
 Similarly you can remove the examples dir from /Applications/Kivy.app/Contents/Resources/kivy/examples/
 or kivyt/tools,  kivy/docs...
 
 This way the whole app can be made to only include the parts that you use inside your app.
 
-You can edit the icons and other settings of your app by editing the YourApp/Contents/info.plist to suit your needs, simply double click this file and make your changes.
+Adjust settings
+^^^^^^^^^^^^^^^
+You can edit the icons and other settings of your app by editing the YourApp/Contents/info.plist to suit your
+needs, simply double click this file and make your changes.
 
-Last step is to make a dmg of your app using the following command::
+Create a DMG
+^^^^^^^^^^^^
+To make a DMG of your app use the following command::
 
-    osx> create-osx-dmg.sh YourApp.app
+    osx> ./create-osx-dmg.sh YourApp.app
 
 This should give you a compressed dmg that will even further minimize the size of your distributed app.
 
 
+Using PyInstaller and Homebrew
+------------------------------
+.. note::
+    - Only use this method if you are an advanced user and know that you need more
+      or newer libraries than Provided by the Kivy.app or Python 3 support.
```

Comment:
if you compile your Kivy.app using kivy-sdk-packager using python 3 you will have a python3 package,  just like with pyinstaller. So Kivy.app method does support python3